app/src/main/cpp/skyline/input/npad_device.h

```diff
@@ -6,6 +6,11 @@
 #include <kernel/types/KEvent.h>
 #include "shared_mem.h"
 
+namespace skyline::constant {
+    constexpr jlong MsInSecond = 1000; //!< The amount of milliseconds in a single second of time
```

Comment:
I think std::chrono can do this

Reply:
It can but I don't think we should explicitly source it from there, similar to how it's done in `common.h` with `NsInSecond` atm

Reply:
Chrono could probably do it in the code itself, no need for a constant then. Seeing as it can also do casts from an arbitrary tick amount we could use for GPU/device timer too

Reply:
I think it actually looks far more unneat with Chrono than it does otherwise, I was using it earlier and explicitly switch to this rather than the other way around. This value is used in the reciprocal and that just doesn't work too well with how `std::chrono` does things and ends up requiring a bunch of casts which looks terrible.